### ISSUE TITLE
Undisclosed vulnerability

### ISSUE THREAD

**Issue by randomstuff** [KaneRoot/dnsmanager#24]
Undisclosed vulnerability.

It would be nice to add support for security issues in order to discuss and track this.

**Comment by KaneRoot** [comment]
I agree, but also I don't have much time for even fixing security issues. Want to share something private with me? Send a mail.
